services/src/integ-test/resources/application-integ.yaml

```diff
@@ -49,4 +49,6 @@ sdt:
     commitInterval: ${PURGE_COMMIT_INTERVAL:1000}
   retry-send:
     cron: ${RETRY_SEND_CRON:0 0 * * * ?}
+  requeue:
+    cron: ${REQUEUE_SCHEDULE_CRON:0 20 0 * * ?}
```

Comment:
The default should be `0 */20 * * * ?`. This means "Run every 20th minute" whereas `0 20 0 * * ?` means "Run at 00:20:00".

Reply:
done